docs/common_tasks.md

```diff
@@ -41,3 +41,10 @@ To add a new repository, the following steps are needed:
 
 For more information on adding a new GitHub repository, see
 [Adding a GitHub repository](submitting_data.md#adding-a-github-repository).
+
+## Debugging Tools
+
```

Comment:
Wondering if this should go in a separate section. Common tasks seems to be a catch-all and might be missed by newcomers.

Reply:
I agree. It's something I've been thinking for a while. We're probably overdue to overhaul the documentation.